ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+  # Main help command
+  parser = argparse.ArgumentParser(
+    description = "USAGE: python post_process_metadata.py --metadata 'data/metadata.tsv'"
```

Comment:
One nice feature of the argparse module is that [it will automatically generate a usage statement for you](https://docs.python.org/3/library/argparse.html#usage) from the arguments you register below. When the user runs your command without any arguments or with `--help` or `-h`, they will get the usage statement.